src/services/utilServices/SendDNSRecordEmailClient.ts

```diff
@@ -0,0 +1,121 @@
+import { groupBy } from "lodash"
+
+export interface DnsRecordsEmailProps {
+  requesterEmail: string
+  repoName: string
+  domainValidationSource: string
+  domainValidationTarget: string
+  primaryDomainSource: string
+  primaryDomainTarget: string
+  redirectionDomainSource?: string
+  redirectionDomainTarget?: string
+}
+
+export interface LaunchFailureEmailProps {
+  // The fields here are optional since a misconfiguration in our
+  // formSG can cause some or even all fields to be missing
+  requesterEmail?: string
+  repoName?: string
+  primaryDomain?: string
+  error: string
+}
+
+const tableStyle =
+  "border-collapse: collapse; width: 100%; border: 1px solid #ddd; text-align: center;"
+
+const thStyle = "padding: 8px; text-align: center; border: 1px solid #ddd;"
+
+const tdStyle = "padding: 8px; text-align: left; border: 1px solid #ddd;"
+
+const headerRowStyle =
+  "background-color: #f2f2f2; border: 1px solid #ddd; text-align: center;"
+
+const repoNameStyle =
+  "padding: 8px; text-align: left; font-weight: bold; border: 1px solid #ddd;"
```

Comment:
Minor/Nit: it'd be better to abstract them to a css style file, together with all other styles in the codebase. 
Understand it might not be done in this PR, but just to share my thoughts.

Reply:
Thanks for the comment! I am inclined to keep at this level of abstraction since this is only used for sending the email in the backend and these styles (as far as I am aware) are only going to be used for this purpose!

Reply:
Sure that makes sense.